src/kem/kyber/arithmetic.rs

```diff
@@ -6,7 +6,7 @@ const BARRETT_SHIFT: i32 = 26;
 const BARRETT_R: i32 = 1i32 << BARRETT_SHIFT;
 const BARRETT_MULTIPLIER: i32 = 20159; // floor((BARRETT_R / FIELD_MODULUS) + 0.5)
 
-#[cfg_attr(hax, hax_lib_macros::requires(value > -BARRETT_R / 2 && value < BARRETT_R / 2))]
+#[cfg_attr(hax, hax_lib_macros::requires(value >= -106527 && value <= 104862))]
```

Comment:
Please add documentation where these numbers are coming from.

Reply:
Tracked in https://github.com/cryspen/libcrux/issues/132